admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -735,3 +735,10 @@ is offline and will not try to connect again for the time specified in **Cache
 Time-To-Live**. If you have a backup server configured Nextcloud will connect to
 it instead. When you have scheduled downtime, check **Disable Main Server**  to
 avoid unnecessary connection attempts.
+
+Note
+----
+
+When a LDAP object's name or surname is left empty, Nextcloud will treat it as an 
```

Comment:
To be more precise, it is about the (configured) display name attribute, by default "displayname".

Reply:
I changed the text, to be more precise.